controllers/velero.go

```diff
@@ -515,6 +516,20 @@ func (r *DPAReconciler) customizeVeleroDeployment(dpa *oadpv1alpha1.DataProtecti
 	if err != nil {
 		return err
 	}
+
+	if dpa.Spec.Configuration.Velero.LogLevel != "" {
+		_, err := logrus.ParseLevel(dpa.Spec.Configuration.Velero.LogLevel)
+		if err != nil {
+			allowedLevels := ""
+			for _, level := range logrus.AllLevels {
+				allowedLevels += level.String() + ","
+			}
+			allowedLevels = strings.TrimSuffix(allowedLevels, ",")
+			return fmt.Errorf("invalid log level %s\nallowed: %s", dpa.Spec.Configuration.Velero.LogLevel, allowedLevels)
```

Comment:
@kaovilai I believe you need to update this error message code block to reflect that `panic` and `fatal` levels are not supported.

Reply:
Thanks. Fixed in https://github.com/openshift/oadp-operator/pull/588/commits/bed9d438854bd16f4f5a60f442be51483c9e3121